beacon-chain/rpc/aggregator/server.go

```diff
@@ -71,7 +79,36 @@ func (as *Server) SubmitAggregateAndProof(ctx context.Context, req *pb.Aggregati
 		return nil, status.Errorf(codes.InvalidArgument, "Validator is not an aggregator")
 	}
 
-	// TODO(3865): Broadcast aggregated attestation & proof via the aggregation topic
+	// Retrieve the aggregated attestation from pool
+	atts := as.AttPool.UnaggregatedAttestations(req.Slot, req.CommitteeIndex)
+
+	// Verify attestations are valid before aggregating and broadcasting them out
+	validAtts := make([]*ethpb.Attestation, 0, len(atts))
+	for _, att := range atts {
+		if err := blocks.VerifyAttestation(ctx, headState, att); err != nil {
+			if err := as.AttPool.DeleteUnaggregatedAttestation(att); err != nil {
+				return nil, status.Errorf(codes.Internal, "Could not delete invalid attestation: %v", err)
+			}
+			continue
+		}
+		validAtts = append(validAtts, att)
+	}
+
+	// Aggregate the attestations and broadcast them
+	aggregatedAtts, err := helpers.AggregateAttestations(validAtts)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Could not aggregate attestations: %v", err)
+	}
+	for _, att := range aggregatedAtts {
+		if att.AggregationBits.Count() > 1 {
```

Comment:
Maybe we can create a public helper func called IsAggregated in some shared package somewhere